exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -22,13 +22,20 @@ function remainderOf(n, d) {
     If you're not sure, step out of the code and use pen + paper. Start
     with simple examples and pay attention to the process you carry out.
   */
+  return console.log(n % d)
```

Comment:
You've got the right idea that `%` is used to calculate the remainder. But you're confused about the difference between a function that _returns_ a value vs. a function that prints a value out to the console. This is probably the #1 thing beginners get tripped up on and it takes a while to see the difference.

Think of a function as a mysterious black box sitting in a field. The black box is going to carry out some work, but we don't necessarily know what. On one side are a place for you to input information. On another side is a place for the results of the black box's work. The inputs and outputs are send/received by writing information on a piece of paper.

So, for example, I write `10` and `2` on two pieces of paper, feed it to the machine, and wait for the machine to do its job. After a while, a piece of paper with the number`5` written on it is printed from the other side.

You could feed this piece of paper to another machine if you wanted.

There is also a kind of loud speaker attached to the box.

`return` is telling the black box to give you the result on a piece of paper. `console.log` is telling the black box to play something over the loud speaker.

We try to explain it, here: https://github.com/jfarmer/intro-javascript/blob/master/Functions/Returning.md